src/datastructures.apib

```diff
@@ -140,3 +140,121 @@
         + timeTracking.deleted
         + timeTracking.updated
         + user.deactivated
+
+## WorkOrderData (object)
++ customer (object, required)
+    + type (enum[string], required)
+        + Members
+            + contact
+            + company
+    + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
++ datetime: `2017-05-26T10:01:49+00:00` (string, required)
++ description (string, optional)
++ department_id: `cef01135-7e51-4f6f-a6eb-6e5e5a885ac7` (string, optional)
++ layout_id: `a05320f6-b3ef-4ff9-a287-2074a12fb3e9` (string, optional)
++ milestone_id: `e94c1363-3426-46a4-921d-fce03f39bbd5` (string, optional)
++ event_id: `9fc14045-5c6c-4ba8-8672-42ea3f26aa63` (string, optional)
```

Comment:
Q : why do we currently need the event id?

Reply:
Providing a meeting ID will mark that meeting as done when the WorkOrder is finished.